packages/framework-provider-kubernetes/src/services/event-registry.ts

```diff
@@ -1,15 +1,13 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
-import { EventEnvelope, Logger } from 'framework-types/dist'
-import { v4 as uuidv4 } from 'uuid';
+import { EventEnvelope, Logger, UUID } from '@boostercloud/framework-types'
```

Comment:
Since we are no longer using uuid dependency, we should remove it from package.json